tendrl/gluster_integration/sds_sync/event_utils.py

```diff
@@ -1,39 +1,36 @@
 import json
 import os
-import socket
 
-from tendrl.commons.event import Event
-from tendrl.commons.message import Message
+from tendrl.commons.utils import log_utils as logger
 from tendrl.commons.utils.time_utils import now as tendrl_now
 
 
-def emit_event(self, resource, curr_value, msg, instance):
+def emit_event(resource, curr_value, msg, instance):
     alert = {}
     alert['source'] = NS.publisher_id
+    alert['classification'] = 'cluster'
     alert['pid'] = os.getpid()
     alert['time_stamp'] = tendrl_now().isoformat()
-    alert['alert_type'] = 'status'
+    alert['alert_type'] = 'STATUS'
     severity = "INFO"
     if curr_value.lower() == "stopped":
```

Comment:
This was valid when we only had volume and brick, Now geo-rep/rebalance/peer is something else right, Can we modify this function to take "severity" as an argument and let caller of this utility fun decide about the severity ?

Reply:
That's a good catch. Would do the required changes.